integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_files_excluded.py

```diff
@@ -0,0 +1,188 @@
+                            "key": key_name
+                        }
+                        ]
+                    }
+                    ],
+                    "copied_to_orca": []
+                },
+                "meta": {
+                    "provider": {
+                    "id": provider_id,
+                    "name": provider_name
+                    },
+                    "collection": {
+                        "meta": {
+                            "orca": {
+                                "excludedFileExtensions":
+                                    excluded_filetype
+                            }
+                        },
+                        "name": collection_name,
+                        "version": collection_version
+                        }
+                },
+                "cumulus_meta": {
+                    "execution_name": execution_id
+                },
+                "task_config": {
+                    "excludedFileExtensions": "{$.meta.collection.meta.orca.excludedFileExtensions}",
+                    "s3MultipartChunksizeMb": "{$.meta.collection.meta.s3MultipartChunksizeMb}",
+                    "providerId": "{$.meta.provider.id}",
+                    "providerName": "{$.meta.provider.name}",
+                    "executionId": "{$.cumulus_meta.execution_name}",
+                    "collectionShortname": "{$.meta.collection.name}",
+                    "collectionVersion": "{$.meta.collection.version}",
+                    "defaultBucketOverride": "{$.meta.collection.meta.orca.defaultBucketOverride}",
+                    "defaultStorageClassOverride": "{$.meta.collection.meta.orca.defaultStorageClassOverride}"
+                },
+                "exception": "None"
+                }
+
+            execution_info = boto3.client("stepfunctions").start_execution(
+                stateMachineArn=helpers.orca_copy_to_archive_step_function_arn,
+                input=json.dumps(copy_to_archive_input, indent=4),
+            )
+
+            step_function_results = helpers.get_state_machine_execution_results(
+                execution_info["executionArn"]
+            )
+
+            self.assertEqual(
+                "SUCCEEDED",
+                step_function_results["status"],
+                "Error occurred while starting step function.",
+            )
+            self.assertEqual(
+                expected_output,
+                json.loads(step_function_results["output"]),
+                "Expected step function output not returned.",
+            )
+
+            # verify that the object does NOT exist in recovery bucket
+            try:
+                head_object_output = boto3.client("s3").head_object(
+                    Bucket=recovery_bucket_name, Key=key_name)
+                if head_object_output["ResponseMetadata"]["HTTPStatusCode"] == 200:
+                    raise Exception(f"{key_name} already exists in {recovery_bucket_name}")
```

Comment:
already exists or was incorrectly ingested to {recovery_bucket_name}

Reply:
updated